app/konnect/mesh-manager/service-mesh.md

```diff
@@ -0,0 +1,88 @@
+
+You now have a very basic {{site.mesh_product_name}} service mesh added to {{site.konnect_short_name}}. This service mesh can only create meshes and policies at the moment, so we must add services and additional configurations to it.
+
+## Install a demo service
+
+Now that you've added a global control plane and a zone to your service mesh in {{site.konnect_short_name}}, you can add services to your mesh. You can use the {{site.mesh_product_name}} Kubernetes demo app to set up four services so you can see how {{site.mesh_product_name}} can be used to control services, monitor traffic, and track resource status.
+
+The demo application consists of four services:
+
+* `frontend`: A web application that lets you browse an online clothing store
+* `backend`: A Node.js API for querying and filtering clothing items
+* `postgres`: A database for storing clothing item reviews
+* `redis`: A data store for the clothing item star ratings
+
+To add the services to your mesh using the demo app, run the following command:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/kumahq/kuma-demo/master/kubernetes/kuma-demo-aio.yaml
+```
+
+You can see the services the Kubernetes demo app added by navigating to **Mesh Manager** in the sidebar of {{site.konnect_short_name}}, selecting the `example-cp` and clicking **Meshes** in the sidebar. You can view the services associated with that mesh by clicking **Default** and the **Services** tab.
+
+For more information about the Kubernetes demo app, see [Explore {{site.mesh_product_name}} with the Kubernetes demo app](/mesh/latest/quickstart/kubernetes/).
+
+## Configure `kumactl` to connect to your global control plane
+
+`kumactl` is a CLI tool that you can use to access {{site.mesh_product_name}}. It can create, read, update, and delete resources in {{site.mesh_product_name}} in Universal/{{site.konnect_short_name}} mode.
+
+You connect `kumactl` to the global control plane in {{site.konnect_short_name}} so that you can run commands against the control plane.
+
+1. From the left navigation menu in {{site.konnect_short_name}}, open {% konnect_icon mesh-manager %} [**Mesh Manager**](https://cloud.konghq.com/mesh-manager) and select the `example-cp` control plane.
+1. Select **Configure kumactl** from the **Global Control Plane Actions** dropdown menu and follow the steps in the wizard to connect `kumactl` to the control plane.
+1. Verify that the services you added from the previous section with the Kubernetes demo app are running correctly:
+```bash
+kumactl get dataplanes
+```
+If your data planes were configured correctly with the demo app, the output should return all four data planes. 
+
+You can now issue commands to your global control plane using `kumactl`. You can see the [`kumactl` command reference](/mesh/latest/generated/cmd/kumactl/kumactl/) for more information about the commands you can use.
```

Comment:
```suggestion
You can now issue commands to your global control plane using `kumactl`. You can see the [`kumactl` command reference](/mesh/latest/generated/cmd/kumactl/kumactl/) for more information about the commands you can use.
```
This URL 404s and not sure if https://deploy-preview-5704--kongdocs.netlify.app/mesh/latest/explore/cli/#kumactl is the correct replacement?

Reply:
Yep! I've just fixed it now